packages/web-components/fast-components/src/design-system-provider/index.ts

```diff
@@ -188,7 +188,7 @@ export class DesignSystemProvider extends FoundationElement {
         converter: swatchConverter,
     })
     @designToken(fillColor)
-    public fillColor: Swatch;
+    public fillColor: Swatch = fillColor.getValueFor(this);
```

Comment:
Doing this is going to cause a lot of un-necessary work for each design-system-provider instance so I don't think we should do this. 

I'd propose updating the property types to allow for undefined, thus not requiring initialization (at least I think that is how it works).  If that *doesn't* work, I'd propose disabling that rule for this file.

Reply:
That should be done 👍